### PR TITLE
LPTOCPCI-319: openshift-pipelines operator on ROSA Hypershift

### PR DESCRIPTION
**Pull request by chetna14manku**

### CODE REVIEW

**Comment by ascerra** [comment]
I'm not sure if I just don't have access to view all users in the openshift-pipelines org but we need to have the openshift ci bots here to pass the check-gh-automation check
https://github.com/orgs/openshift-pipelines/people

An easy workaround for this is to not use config/openshift-pipelines/release-tests to hold this config but instead just use config/rhpit/interop-tests to hold the config.

File: ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.11__openshift-pipelines-hypershift-4.13-lp-rosa-hypershift.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: openshift-pipelines-rosa-hypershift
+  cron: 0 10 * * 1
+  remote_api: true
```
Comment (ascerra) [inline ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.11__openshift-pipelines-hypershift-4.13-lp-rosa-hypershift.yaml:27]: We can remove this line

**Comment by chetna14manku** [comment]
> I'm not sure if I just don't have access to view all users in the openshift-pipelines org but we need to have the openshift ci bots here to pass the check-gh-automation check https://github.com/orgs/openshift-pipelines/people
> 
> An easy workaround for this is to not use config/openshift-pipelines/release-tests to hold this config but instead just use config/rhpit/interop-tests to hold the config.

Hi @ascerra Openshift bots are installed in [openshift-pipelines/release-tests](https://github.com/openshift-pipelines/release-tests/) repo, but doesnot have `admin` role, I have already asked the team to update the permissions.
If it is not done (by Friday afternoon), I'm ready to move it to rhpit/interop-tests.

File: ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.11__openshift-pipelines-hypershift-4.13-lp-rosa-hypershift.yaml (hunk 1 above)
Comment (chetna14manku) [reply ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.11__openshift-pipelines-hypershift-4.13-lp-rosa-hypershift.yaml]: removed

**Comment by ascerra** [comment]
> 

I believe they need to be installed on the Github Org not just the repo

**Comment by chetna14manku** [comment]
> > 
> 
> I believe they need to be installed on the Github Org not just the repo

@ascerra there are some [existing configs](https://github.com/openshift/release/tree/master/ci-operator/config/openshift-pipelines) for openshift-pipelines org, so I believed the bots should already be installed on the Guthub Org. But still the gh-automation failed in this PR.

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-openshift-pipelines-hypershift-4.13-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-openshift-pipelines-hypershift-4.13-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Pull request merged**